misc/userscripts/readability-js

```diff
@@ -54,6 +54,35 @@ const HEADER = `
         figure img {
             display: block;
         }
+        table,
+        th,
+        td {
+            border: 1px solid currentColor;
```

Comment:
TIL about `currentColor` - nice!

Reply:
This isn't a real problem right? I wasn't fully aware of what `currentColor` did when I copied it over, but it may potentially make the user defined text color (e.g. one set through a stylesheet option) congruent with the color of tables.